sdk/boards/sonata-0.2.json

```diff
@@ -75,6 +75,7 @@
         "ipconfigDRIVER_INCLUDED_RX_IP_CHECKSUM=1",
         "ipconfigDRIVER_INCLUDED_TX_IP_CHECKSUM=1"
     ],
+    "cflags": "-enable-machine-outliner=never",
```

Comment:
I *think* this is not needed on 0.2.  I'll check, but I believe 0.2 has an old enough version of Ibex that it doesn't introduce the problem that the latest one fixes.

Reply:
You had previously said it was needed on 0.2 here: https://github.com/CHERIoT-Platform/cheriot-sail/issues/85#issuecomment-2517482580

I'm happy to do whatever, but I don't know how to find the correct answer.